### PR TITLE
Readme spanish translation

### PR DESCRIPTION
**Pull request by amendezm**
Related to #345 

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Readme translations:
 <kbd>[<img title="Hebrew" alt="Hebrew" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/il.svg" width="22">](docs/README-he.md)</kbd>
 <kbd>[<img title="Italiano" alt="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">](docs/README-it.md)</kbd>
 <kbd>[<img title="Türkçe" alt="Türkçe" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/tr.svg" width="22">](docs/README-tr.md)</kbd>
+<kbd>[<img title="Türkçe" alt="Türkçe" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/es.svg" width="22">](docs/README-es.md)</kbd>
```
Comment (nukeop) [inline README.md:33]: Can you change the title and alt?

Comment (amendezm) [reply README.md:33]: Sure!

**Comment by amendezm** [comment]
@nukeop done!

**Comment by nukeop** [comment]
Great, thank you.

**Pull request merged**